en/micro-integrator/docs/references/connectors/db-event-inbound-endpoint/db-event-inbound-endpoint-example.md

```diff
@@ -0,0 +1,139 @@
+        <parameter name="connectionValidationQuery">SELECT 1</parameter>
+        <parameter name="registryPath">dbEventIE/timestamp</parameter>
+    </parameters>
+</inboundEndpoint>
+```
+
+
+## Exporting Integration Logic as a CApp
+
+**CApp (Carbon Application)** is the deployable artefact on the Enterprise Integrator runtime. Let us see how we can export integration logic we developed into a CApp. To export the `Solution Project` as a CApp, a `Composite Application Project` needs to be created. Usually, when a solution project is created, this project is automatically created by Integration Studio. If not, you can specifically create it by navigating to  **File** -> **New** -> **Other** -> **WSO2** -> **Distribution** -> **Composite Application Project**. 
+
+1. Right click on Composite Application Project and click on **Export Composite Application Project**.</br> 
+  <img src="/assets/img/connectors/capp-project1.png" title="Export as a Carbon Application" width="300" alt="Export as a Carbon Application" />
+
+2. Select an **Export Destination** where you want to save the .car file. 
+
+3. In the next **Create a deployable CAR file** screen, select inbound endpoint and sequence artifacts and click **Finish**. The CApp will get created at the specified location provided at the previous step. 
+
+
+
+## Deploying on WSO2 Enterprise Integrator
+
+1. Navigate to [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for `DB Event Listener`. Click on `DB Event Listener` and download the jar file by clicking on `Download Inbound Endpoint`. Copy this jar file into  <PRODUCT-HOME>/lib folder. 
+
+2. Download `mysql-connector-java` associated with `MySQL` server and add it to <PRODUCT-HOME>/lib folder.
+
+3. Copy the exported carbon application to the <PRODUCT-HOME>/repository/deployment/server/carbonapps folder. 
+
+4. Start the server 
+
+
+Now WSO2 EI will start listening to the data changes of `CDC_CUSTOM` table. 
+
+
+
+## Testing
+
+### Adding a new record
+
+1. Using MySQL terminal execute following SQL to insert a new customer record into the table. 
+```sql
+INSERT INTO `test`.`CDC_CUSTOM` (`ID`, `NAME`, `ADDRESS`, `AMOUNT`) VALUES (001, "john", "22/3, Tottenham Court, London" , 1000);
+```
+2. You should get a log in wso2 server console as below. 
+```
+[2020-03-26 17:40:00,871]  INFO {org.apache.synapse.mediators.builtin.LogMediator} - To: , MessageID: urn:uuid:4B1D55C3ABCEE82B961585224600739, Direction: request, message = event received, Envelope: <?xml version='1.0' encoding='utf-8'?><soapenv:Envelope xmlns:soapenv="http://www.w3.org/2003/05/soap-envelope"><soapenv:Body><Record><ID>1</ID><NAME>john</NAME><ADDRESS>22/3, Tottenham Court, London</ADDRESS><AMOUNT>1000</AMOUNT><PAID>false</PAID><UPDATED_AT>2020-03-26 16:57:57.0</UPDATED_AT></Record></soapenv:Body></soapenv:Envelope>
+```
+3. If you add another new record only that new record will get notified into WSO2 EI.
+
+
+### Update an existing record 
```

Comment:
Same comments from the previous section apply here as well.

Reply:
Applied.